tests/cypress/support/const.js

```diff
@@ -33,6 +33,17 @@ export const multiAttrParams = {
     values: 'Attr value 2',
     type: 'Text',
 };
+export const skeleton = {
+    name: 'skeleton',
+    type: 'shape',
+    points: [
+        { x: 0.55, y: 0.15 },
+        { x: 0.20, y: 0.35 },
+        { x: 0.43, y: 0.55 },
+        { x: 0.63, y: 0.38 },
+        { x: 0.27, y: 0.15 },
+    ],
+};
```

Comment:
I do not think we want to add skeleton to each test, it increases running time and quite useless in the most of cases

Reply:
And if you would like to improve the tests here, I believe creating a task "by default' should be headless, it is much faster and this is good cypress pattern. 
Of course we still need some tests to create tasks via interface, but they should be a different tests that checks exactly tasks creating pipeline.

Reply:
Removed that. 
Ive spent a couple of hours trying to headless-create task with .zip client files, but had no luck(strangely uploading .png files works well). 
Lets keep everythin as it is here for now, and keep that improvement as separate PR as well